cogs/greetings.py

```diff
@@ -55,7 +53,7 @@ async def on_raw_reaction_add(self, payload):
         if payload.message_id == self.welcome_message_id:
             if payload.channel_id == WELCOME_CHANNEL_ID:
                 if WELCOME_REACT_EMOJI in str(payload.emoji):
-                    guild = self.client.get_guild(payload.guild_id)
+                    guild = self.bot.get_guild(payload.guild_id)
                     member = guild.get_member(payload.user_id)
                     if member is not None and not member.bot:
```

Comment:
Change check to `if member and not member.bot` or `if member != None and not member.bot` because `is` keyword is different than `== or !=`